src/scripts/main.js

```diff
@@ -2,7 +2,20 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard');
 
-// write your code here
+const newPeople = [...people];
+
+newPeople.forEach(person => {
+  const cells = document.createElement('tr');
```

Comment:
```suggestion
  const tableRow = document.createElement('tr');
```
Actually, it's a row :) So let's name it correspondingly